x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/authz/ReadActionsTests.java

```diff
@@ -38,9 +39,14 @@
 
 public class ReadActionsTests extends SecurityIntegTestCase {
 
+    @After
+    public void cleanupSecurityIndex() {
+        super.deleteSecurityIndex();
```

Comment:
Need to cleanup `.security` index to avoid tripping the following assertion:
`java.lang.AssertionError: FixedBitSet cache size must be 0 on node: `